pages/watches-support.json

```diff
@@ -634,6 +650,7 @@
       "soc":"msm8226",
       "kernelversion":"3.10",
       "androidversion":"marshmallow",
+      "releasedate": 2015,
```

Comment:
Maybe it's worth removing the "2015" from the model name here ("Moto 360 2015") ?